app/actions/session.js

```diff
@@ -14,7 +14,7 @@ export const setUser = createAction(SET_USER);
 export const removeUser = createAction(REMOVE_USER);
 
 export const signinUser = user =>
-  async dispatch => {
+  async (dispatch) => {
     dispatch(loadData());
```

Comment:
why???

Reply:
There is only one case for not using brackets around params: if there is only one param and arrow function haven't explicit `return` keyword (arrow function with immediately return value). In other cases brackets are needed.